web/modules/custom/dbcdk_community/src/Form/ProfileEditForm.php

```diff
@@ -24,79 +24,50 @@
 class ProfileEditForm extends FormBase implements ContainerInjectionInterface {
 
   /**
-   * The DBCDK Community Service Profile API.
+   * The profile repository to use.
    *
-   * @var \DBCDK\CommunityServices\Api\ProfileApi $profileApi
+   * @var \Drupal\dbcdk_community\Profile\ProfileRepository $profileRepository
    */
-  protected $profileApi;
+  protected $profileRepository;
 
   /**
    * The Community Service Profile object from the username context.
    *
-   * @var \DBCDK\CommunityServices\Model\Profile $profile
+   * @var Profile $profile
    */
   protected $profile;
 
   /**
-   * Array of all defined community roles.
-   *
-   * @var \DBCDK\CommunityServices\Model\CommunityRole[] $communityRoles
-   */
-  protected $communityRoles;
-
-  /**
    * Creates a Profile Edit Form instance.
    *
-   * @param \DBCDK\CommunityServices\Api\ProfileApi $profile_api
-   *   The DBCDK Community Service Profile API.
-   * @param \DBCDK\CommunityServices\Model\Profile $profile
+   * @param \Drupal\dbcdk_community\Profile\ProfileRepository $profile_repository
+   *   The profile repository to use.
+   * @param \DBCDK\CommunityServices\Model\Profile|\Drupal\dbcdk_community\Profile\Profile $profile
```

Comment:
When I glans through the code in this class, it assumes that it's a `\Drupal\dbcdk_community\Profile\Profile` that has been passed (ex: it assumes that it can fetch roles with the [`Profile::getCommunityRoles`](https://github.com/DBCDK/biblo-admin/pull/67/files#diff-3fb04a6aef57ae4c52468bf184cacdebR135) method. And from what I can see, then the Repository will always [return an "enriched" profile or an empty result](https://github.com/DBCDK/biblo-admin/blob/feature/refactor-profiles/web/modules/custom/dbcdk_community/src/Profile/ProfileRepository.php#L136).

If this is the case and the \DBCDK\CommunityServices\Model\Profile is removed from the documentation, then I feel like the `protected $profile` property should have a fully declared namespace as well `\Drupal\dbcdk_community\Profile\Profile`.